{{cookiecutter.project_slug}}/Makefile

```diff
@@ -44,6 +45,9 @@ HTTP_AUTH?=--http-auth
 # TRAINING_COMMAND="bash -c 'cd $(PROJECT_PATH_ENV) && python -u $(CODE_DIR)/train.py --data $(DATA_DIR)'"
 TRAINING_COMMAND?='echo "Replace this placeholder with a training script execution"'
 
+# Port for remote debugging via PyCharm Pro
```

Comment:
Let's remove PyCharm from here. We'll provide information in docs.

Reply:
ok thanks